### ISSUE TITLE
Startup crash with v2.3.1

### ISSUE THREAD

**Issue by mattsch** [maddox/harmony-api#47]
After updating to 2.3.1 and running npm update, I'm getting the following error:
```
Dec 01 14:07:55 ha node[24016]: Starting discovery.
Dec 01 14:07:55 ha node[24016]: /home/pi/harmony-api/app.js:101
Dec 01 14:07:55 ha node[24016]: activitySlug = harmonyHubStates[hubSlug].current_activity.slug
Dec 01 14:07:55 ha node[24016]: ^
Dec 01 14:07:55 ha node[24016]: TypeError: Cannot read property 'current_activity' of undefined
Dec 01 14:07:55 ha node[24016]: at MqttClient.<anonymous> (/home/pi/harmony-api/app.js:101:45)
Dec 01 14:07:55 ha node[24016]: at emitThree (events.js:97:13)
Dec 01 14:07:55 ha node[24016]: at MqttClient.emit (events.js:175:7)
Dec 01 14:07:55 ha node[24016]: at MqttClient._handlePublish (/home/pi/harmony-api/node_modules/mqtt/lib/client.js:792:12)
Dec 01 14:07:55 ha node[24016]: at MqttClient._handlePacket (/home/pi/harmony-api/node_modules/mqtt/lib/client.js:282:12)
Dec 01 14:07:55 ha node[24016]: at process (/home/pi/harmony-api/node_modules/mqtt/lib/client.js:238:12)
Dec 01 14:07:55 ha node[24016]: at MqttClient._handlePacket (/home/pi/harmony-api/node_modules/mqtt/lib/client.js:290:7)
Dec 01 14:07:55 ha node[24016]: at process (/home/pi/harmony-api/node_modules/mqtt/lib/client.js:238:12)
Dec 01 14:07:55 ha node[24016]: at Writable.writable._write (/home/pi/harmony-api/node_modules/mqtt/lib/client.js:248:5)
Dec 01 14:07:55 ha node[24016]: at doWrite (/home/pi/harmony-api/node_modules/mqtt/node_modules/readable-stream/lib/_stream_writable.js:237:10)

```

Reverting back to 2.1.0 fixes things.

**Comment by maddox** [comment]
Fixed in #48. 

2.3.2 released.

**Comment by mattsch** [comment]
Thanks!
